tests/test_package_changes.py

```diff
@@ -1,65 +1,76 @@
+from asu.build_request import BuildRequest
 from asu.package_changes import appy_package_changes
 
 
 def test_apply_package_changes_adds_kmod_switch_rtl8366s():
-    req = {
-        "version": "23.05",
-        "target": "ath79/generic",
-        "profile": "buffalo_wzr-hp-g300nh-s",
-        "packages": ["kmod-ath9k-htc"],
-    }
-    appy_package_changes(req)
+    build_request = BuildRequest(
+        **{
```

Comment:
For these, you could also say
```
... = BuildRequest(
    version = "23.05",
    target = "ath79/generic",
    ...
)
```
Although the current form is more suggestive of the json that it represents, so...